.github/workflows/query-engine-driver-adapters.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       QUERY_BATCH_SIZE: '10'
       WORKSPACE_ROOT: ${{ github.workspace }}
 
-    runs-on: ubuntu-latest
+    runs-on: buildjet-8vcpu-ubuntu-2004
```

Comment:
Is this change intended?

Reply:
Yes, temporarily. It's subject of each own commit.

Reply:
Once this is ready to be merged, I will remove that.

Reply:
```suggestion
    runs-on: ubuntu-latest
```